definitions/npm/http-status-codes_v1.x.x/test_http-status-codes_v1.x.x.js

```diff
@@ -0,0 +1,23 @@
+import { describe, it } from 'flow-typed-test';
```

Comment:
Please move the test file to be paired with the flow version. Sometimes tests need to vary based on implementation, so it's standard practice to keep the tests versioned as well.

Reply:
Thanks to reviewing :pray:
And I've fixed it at f21726dd08297e2cd1e3535b1cf642b58ffecded